src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Shipped.txt

```diff
@@ -600,6 +600,7 @@ NuGet.Commands.RestoreArgs.FallbackRuntimes.get -> System.Collections.Generic.Ha
 NuGet.Commands.RestoreArgs.FallbackRuntimes.set -> void
 NuGet.Commands.RestoreArgs.GetEffectiveFallbackPackageFolders(NuGet.Configuration.ISettings settings) -> System.Collections.Generic.IReadOnlyList<string>
 NuGet.Commands.RestoreArgs.GetEffectiveGlobalPackagesFolder(string rootDirectory, NuGet.Configuration.ISettings settings) -> string
+NuGet.Commands.RestoreArgs.GetForceUpdateLastAccessTime(NuGet.Configuration.ISettings settings) -> bool
```

Comment:
I think these need to be in the unshipped file; https://github.com/NuGet/NuGet.Client/blob/dev/docs/nuget-sdk.md#development

Reply:
Yep, my bad. Fixed.